cmd/config-manager/main.go

```diff
@@ -394,18 +394,18 @@ func updateSymlink(config string, f *Flags) (bool, error) {
 		src = filepath.Join(f.ConfigFileSrcdir, config)
 	}
 
-	exists, err := fileExists(f.ConfigFileDst)
+	exists, err := symlinkExists(f.ConfigFileDst)
 	if err != nil {
-		return false, fmt.Errorf("error checking if file '%s' exists: %v", f.ConfigFileDst, err)
+		return false, fmt.Errorf("error checking if symlink '%s' exists: %v", f.ConfigFileDst, err)
 	}
 	if exists {
 		srcRealpath, err := filepath.EvalSymlinks(src)
-		if err != nil {
+		if err != nil && !os.IsNotExist(err) {
```

Comment:
Since we're creating a symlink pointing at `src` here, does that not mean that non-existence is an error too?

Reply:
It's been quite a long time and I honestly can't remember why I put that condition. 

Reviewing it now, I agree it doesn't make much sense as we want to check whether the file referenced by the link exists. I'm pushing a commit to remove it